python/pyarrow/parquet/core.py

```diff
@@ -3038,6 +3065,7 @@ def read_table(source, *, columns=None, use_threads=True, metadata=None,
                 partitioning=partitioning,
                 coerce_int96_timestamp_unit=coerce_int96_timestamp_unit,
                 use_legacy_dataset=True,
+                page_checksum_verification=page_checksum_verification,
```

Comment:
I think this can be removed as we check that `page_checksum_verification` must equal the default (`False`) for the legacy behaviour with your last changes:
https://github.com/apache/arrow/blob/74341e3e0510d4c7fb2793aafae55c95f754a139/python/pyarrow/parquet/core.py#L3045-L3048

Reply:
Thanks! I integrated the fix